.github/workflows/add-label-when-promoted.yaml

```diff
@@ -65,7 +65,7 @@ jobs:
             echo "backport_label=false" >> $GITHUB_OUTPUT
           fi
       - name: Run auto-backport.py when label was added
-        if: ${{ github.event_name == 'pull_request_target' && steps.check_label.outputs.backport_label == 'true' && github.event.pull_request.merged == true }}
+        if: ${{ github.event_name == 'pull_request_target' && steps.check_label.outputs.backport_label == 'true' && github.event.pull_request.state == 'closed' }}
```

Comment:
We can have both: remember, multi-commit PRs get closed as "merged" (nice purple color), while single-commit ones are closed (red). We need to support both.

Reply:
state  is valid for both `merged` and `closed` PRs , while `merged` is valid for merge only

So we should use `state`

Reply:
just wanted to clarify the "state" field of a `pull_request`, i am pasting the related part of the response schema definition of `/repo/{owner}/{repo}/pulls/{pull_number}` from https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#get-a-pull-request:

```yaml
    "state": {
      "description": "State of this Pull Request. Either `open` or `closed`.",
      "enum": [
        "open",
        "closed"
      ],
      "type": "string",
      "examples": [
        "open"
      ]
    },
```

so the "state" of a given pull request can only be "open" or "closed". and the latter applies to both of our use cases -- "merged" and "closed".